### ISSUE TITLE
Unsupported Payment Methods

### ISSUE THREAD

**Issue by mrortach** [LearnPress/learnpress#567]
MaxCoach, I bought your theme a long time ago but I never had the time to create content. Now I'm trying again, but the payment methods do not work in my country.

Paid Memberships Pro: Stripe, Paypal... These are not supported in your country.
Supported:
Iyzico

**Comment by tungnxt89** [comment]
Hi MaxCoach,

You country is Turkey? we'll check Iyzico payment and try integrate it for LearnPress. We see wordpress has addon for Woo https://wordpress.org/plugins/iyzico-woocommerce/#description. You can try it if you use addon LearnPess Woocommerce.

Thanks.

**Comment by mrortach** [comment]
Yes, For now, I am using Knit Pay integrated with iyzico. It would be nice if it was integrated.
